oio/account/backend.py

```diff
@@ -918,9 +921,16 @@ def refresh_bucket(self, bucket_name, **kwargs):
                 self.clistkey(account_id), self.bkey(bucket_name),
                 self.blistkey(account_id)]
         args = [bucket_name]
+
         try:
-            self.script_refresh_bucket(
-                keys=keys, args=args)
+            values = self.script_refresh_bucket(keys=keys, args=args,
+                                                client=self.conn_slave)
+            self.conn.hmset(self.bkey(bucket_name), {
```

Comment:
The purpose of the "bucket refresh" operation is to fix incoherences in the bucket stats. But this 2-step operation does not guarantee that the count will be coherent after its execution.